src/billing/components/Free/PAYGConversion.tsx

```diff
@@ -67,15 +67,17 @@ export const Credit250PAYGConversion: FC = () => {
           <ul className="credit-250-conversion-panel--benefits">
             <li>Unlimited buckets to store your data</li>
             <li>Unlimited storage retention</li>
-            <li>Unlimited dashboards</li>
-            <li>Unlimited tasks</li>
-            <li>Unlimited alert checks and notification rules</li>
-            <li>HTTP and PagerDuty notifications</li>
             {!orgUsesIOx && (
-              <li>
-                Up to {Intl.NumberFormat().format(CARDINALITY_LIMIT)} series
-                cardinality
-              </li>
+              <>
+                <li>Unlimited dashboards</li>
```

Comment:
Does the absence of limits (no task limit, no alert limit, no pagerduty limit) only apply to TSM?

Reply:
Only TSM should get this message, I can see how that isn't very clear. I can restate it as a ternary on `orgUsesIOx`.